### PR TITLE
apache-httpd: adding support for sslServerChain

### PR DESCRIPTION
**Pull request by ecraven**
This small patch adds support for the SSLCertificateChainFile apache option.

### CODE REVIEW

File: nixos/modules/services/web-servers/apache-httpd/per-server-options.nix [hunk 1]
```diff
@@ -56,6 +56,12 @@ with lib;
     description = "Path to server SSL certificate key.";
   };
 
+  sslServerChain = mkOption {
+    type = types.path;
+    example = "/var/ca.pem";
+    description = "Path to server SSL extra chain file.";
```
Comment (edolstra) [inline nixos/modules/services/web-servers/apache-httpd/per-server-options.nix]: Why "extra" chain file?

**Comment by ecraven** [comment]
The certificate I use needs an extra intermediate chain file.
Would you prefer not to make the SSLCertificateChainFile apache option available at all? It's just easier than using extraConfig, for me.

**Comment by ecraven** [comment]
removed the word "extra", sorry, didn't understand what you meant at first

**Comment by ecraven** [comment]
thanks for merging this!

**Pull request merged**